invenio/ext/iiif/__init__.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""
+    invenio.ext.iiif
+    ----------------
+
+    This module provides initialization and configuration for `flask_iiif`
+    module.
+"""
+from flask_iiif import IIIF
+from flask_iiif.errors import MultimediaImageNotFound
+
+from ..cache import cache
+from invenio.modules.documents.api import Document
```

Comment:
Global `invenio` imports might cause troubles. I would move them inside `uuid_to_path` function that should be probably defined inside `invenio.modules.documents.utils/wrappers`.